tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7676,6 +7676,9 @@ def err_hlsl_incorrect_bind_semantic: Error< // Is an error in fxc in some cases
 def warn_hlsl_unary_negate_unsigned : Warning<
   "unary negate of unsigned value is still unsigned">,
   InGroup<Conversion>, DefaultWarn;
+def warn_hlsl_impcast_gl_mismatch : Warning<
```

Comment:
The precedent for the abbreviation of `globallycoherent` is `glc` which I think would be preferred over `gl` here.

Reply:
Fixed in 64bf4c5d873ec447afecd2d6574398becbb425b6